components/builder-originsrv/src/data_store.rs

```diff
@@ -711,6 +711,38 @@ impl DataStore {
         Ok(response)
     }
 
+    pub fn list_origin_packages_for_version(
+        &self,
+        opl: &originsrv::OriginPackagesForVersionRequest,
```

Comment:
It was culted from a similar function, yeah. Req is clearer, I'll use that. Thanks for spotting it.